V2/Makefile

```diff
@@ -0,0 +1,15 @@
+install:
+		@echo "Installing Time Tracker"
+		npm install
+		pip install --upgrade pip
+		pip install -r requirements.txt
+		@echo "Completed! "
+
+deploy:
```

Comment:
does it deploy to a upper or lower environment?

Reply:
at this point it deploys to prod